castle/cms/tests/test_crawl.py

```diff
@@ -51,7 +51,7 @@ def test_crawl_page(self):
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
-<meta name="generator" content="Plone - http://plone.org"/>
+<meta name="generator" content="Castle - http://www.wildcardcorp.com"/>
```

Comment:
I know it seems picky but we want to say CastleCMS as opposed to just Castle wherever it appears hear, we've tried to maintain that name through out the codebase